.github/workflows/buildkit.yml

```diff
@@ -70,8 +70,8 @@ jobs:
         run: |
           ./hack/go-mod-prepare.sh
           # FIXME(thaJeztah) temporarily overriding version to use for tests; remove with the next release of buildkit
-          # echo "BUILDKIT_REF=$(./hack/buildkit-ref)" >> $GITHUB_ENV
-          echo "BUILDKIT_REF=e27c8e24bb9ee92a170567b8b597201925ae9b8a" >> $GITHUB_ENV
+          echo "BUILDKIT_REF=$(./hack/buildkit-ref)" >> $GITHUB_ENV
+          # echo "BUILDKIT_REF=e27c8e24bb9ee92a170567b8b597201925ae9b8a" >> $GITHUB_ENV
```

Comment:
Have 3 failures in BuildKit tests; looks like those are fixed by https://github.com/moby/buildkit/pull/3275, but not yet backported to v0.10;

```
2022-11-14T09:50:09.9335460Z === CONT  TestIntegration/TestBuildInfoDepsMainNoSource/worker=dockerd/frontend=builtin
2022-11-14T09:50:11.3813682Z     dockerfile_buildinfo_test.go:868: 
2022-11-14T09:50:11.3814123Z         	Error Trace:	dockerfile_buildinfo_test.go:868
2022-11-14T09:50:11.3814411Z         	            				run.go:88
2022-11-14T09:50:11.3814704Z         	            				run.go:202
2022-11-14T09:50:11.3815135Z         	Error:      	Not equal: 
2022-11-14T09:50:11.3815533Z         	            	expected: "sha256:419455202b0ef97e480d7f8199b26a721a417818bc0e2d106975f74323f25e6c"
2022-11-14T09:50:11.3815975Z         	            	actual  : "sha256:e5af37e16b3ed9f4d3afca28e628c4f15c9053a902f80bec2c1a6109a7a1ae6c"
2022-11-14T09:50:11.3816355Z         	            	
2022-11-14T09:50:11.3834210Z         	            	Diff:
2022-11-14T09:50:11.3834666Z         	            	--- Expected
2022-11-14T09:50:11.3834874Z         	            	+++ Actual
2022-11-14T09:50:11.3835147Z         	            	@@ -1 +1 @@
2022-11-14T09:50:11.3835559Z         	            	-sha256:419455202b0ef97e480d7f8199b26a721a417818bc0e2d106975f74323f25e6c
2022-11-14T09:50:11.3835923Z         	            	+sha256:e5af37e16b3ed9f4d3afca28e628c4f15c9053a902f80bec2c1a6109a7a1ae6c
2022-11-14T09:50:11.3836567Z         	Test:       	TestIntegration/TestBuildInfoDepsMainNoSource/worker=dockerd/frontend=builtin
2022-11-14T09:50:11.3992936Z     sandbox.go:235: stdout: /usr/bin/dockerd
2022-11-14T09:50:11.3993239Z     sandbox.go:235: stderr: /usr/bin/dockerd
2022-11-14T09:53:48.8533481Z --- FAIL: TestIntegration (1.67s)
2022-11-14T09:53:48.8613277Z     --- FAIL: TestIntegration/TestBuildInfoDepsMainNoSource/worker=dockerd/frontend=builtin (1.47s)

2022-11-14T09:53:48.8616184Z     --- FAIL: TestIntegration/TestBuildInfoDeps/worker=dockerd/frontend=builtin (1.73s)
2022-11-14T09:50:14.4528348Z === CONT  TestIntegration/TestBuildInfoDeps/worker=dockerd/frontend=builtin
2022-11-14T09:50:16.1624171Z     dockerfile_buildinfo_test.go:595: 
2022-11-14T09:50:16.1624510Z         	Error Trace:	dockerfile_buildinfo_test.go:595
2022-11-14T09:50:16.1624767Z         	            				run.go:88
2022-11-14T09:50:16.1624964Z         	            				run.go:202
2022-11-14T09:50:16.1625177Z         	Error:      	Not equal: 
2022-11-14T09:50:16.1625501Z         	            	expected: "sha256:419455202b0ef97e480d7f8199b26a721a417818bc0e2d106975f74323f25e6c"
2022-11-14T09:50:16.1625927Z         	            	actual  : "sha256:e5af37e16b3ed9f4d3afca28e628c4f15c9053a902f80bec2c1a6109a7a1ae6c"
2022-11-14T09:50:16.1626220Z         	            	
2022-11-14T09:50:16.1626413Z         	            	Diff:
2022-11-14T09:50:16.1626867Z         	            	--- Expected
2022-11-14T09:50:16.1627064Z         	            	+++ Actual
2022-11-14T09:50:16.1627328Z         	            	@@ -1 +1 @@
2022-11-14T09:50:16.1627722Z         	            	-sha256:419455202b0ef97e480d7f8199b26a721a417818bc0e2d106975f74323f25e6c
2022-11-14T09:50:16.1628087Z         	            	+sha256:e5af37e16b3ed9f4d3afca28e628c4f15c9053a902f80bec2c1a6109a7a1ae6c
2022-11-14T09:50:16.1628478Z         	Test:       	TestIntegration/TestBuildInfoDeps/worker=dockerd/frontend=builtin
2022-11-14T09:50:16.1722127Z     sandbox.go:235: stdout: /usr/bin/dockerd
2022-11-14T09:50:16.1722419Z     sandbox.go:235: stderr: /usr/bin/dockerd

2022-11-14T09:53:48.8625563Z     --- FAIL: TestIntegration/TestBuildInfoSources/worker=dockerd/frontend=builtin (1.95s)
2022-11-14T09:50:21.0743797Z === CONT  TestIntegration/TestBuildInfoSources/worker=dockerd/frontend=builtin
2022-11-14T09:50:23.0094854Z     dockerfile_buildinfo_test.go:131: 
2022-11-14T09:50:23.0096830Z         	Error Trace:	dockerfile_buildinfo_test.go:131
2022-11-14T09:50:23.0097133Z         	            				run.go:88
2022-11-14T09:50:23.0097346Z         	            				run.go:202
2022-11-14T09:50:23.0097569Z         	Error:      	Not equal: 
2022-11-14T09:50:23.0097892Z         	            	expected: "sha256:419455202b0ef97e480d7f8199b26a721a417818bc0e2d106975f74323f25e6c"
2022-11-14T09:50:23.0098298Z         	            	actual  : "sha256:e5af37e16b3ed9f4d3afca28e628c4f15c9053a902f80bec2c1a6109a7a1ae6c"
2022-11-14T09:50:23.0098601Z         	            	
2022-11-14T09:50:23.0098799Z         	            	Diff:
2022-11-14T09:50:23.0099245Z         	            	--- Expected
2022-11-14T09:50:23.0099451Z         	            	+++ Actual
2022-11-14T09:50:23.0099710Z         	            	@@ -1 +1 @@
2022-11-14T09:50:23.0100116Z         	            	-sha256:419455202b0ef97e480d7f8199b26a721a417818bc0e2d106975f74323f25e6c
2022-11-14T09:50:23.0100485Z         	            	+sha256:e5af37e16b3ed9f4d3afca28e628c4f15c9053a902f80bec2c1a6109a7a1ae6c
2022-11-14T09:50:23.0100868Z         	Test:       	TestIntegration/TestBuildInfoSources/worker=dockerd/frontend=builtin
2022-11-14T09:50:23.0166989Z     sandbox.go:235: stderr: /usr/bin/dockerd
```